package/MDAnalysis/analysis/base.py

```diff
@@ -73,10 +75,13 @@ def _setup_frames(self, trajectory, start=None,
         self.stop = stop
         self.step = step
         self.nframes = len(range(start, stop, step))
+        if self.nframes > 0:
+            self.percentage = ProgressMeter(self.nframes, interval=10, format=
+        "Fitted frame %(step)5d/%(numsteps)d  [%(percentage)5.1f%%]\r")
```

Comment:
actually it might be better to do this in another PR. Reviewing 2 things in one makes it easier to miss things.